src/api.ts

```diff
@@ -0,0 +1,32 @@
+const BASE_URL = 'http://localhost:3000/api';
+
+export interface Todo {
+  id: string;
+  title: string;
+}
```

Comment:
I think `_id` and `text` property is missing here. There are eslint errors on the home page.

![image](https://github.com/jem-vananaz/buddy-todo-app/assets/132255481/97a49522-82d6-4b1c-ac1d-8ad0218581b5)